platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -71,6 +71,34 @@ Starting with Gradle 9.0, the Gradle API is annotated using JSpecify instead.
 
 For more details and potential breakages, see the dedicated [upgrading guide section](userguide/upgrading_version_8.html).
 
+### Task graph diagnostic
+
+We introduce a new task graph diagnostic feature to help you understand the task graph of your build.
```

Comment:
```suggestion
Gradle 9.0.0 introduces a new task graph diagnostic feature to help you visualize the task execution plan without running any tasks.
```

Reply:
Note: tasks from build logic still can run, so "without running any tasks" is misleading